BadHabbit/BadHabbit.xcodeproj/project.pbxproj

```diff
@@ -111,6 +117,7 @@
 		E58D1BC02A1D000000137CC6 /* DesignKit */ = {
 			isa = PBXGroup;
 			children = (
+				F645559B2A62D0F20070947C /* Fonts */,
```

Comment:
shouldn't be in DesignKit. Design Kit only for swift files. Support Files right place for it

Reply:
changed